src/github_runner_image_builder/builder.py

```diff
@@ -214,6 +217,15 @@ def run(arch: Arch, base_image: BaseImage, runner_version: str) -> None:
     logger.info("Compressing image.")
     _compress_image(base_image_path)
 
+    image = store.upload_image(
```

Comment:
Moved uploading image from cli to run. This is to match the behavior of external builder and chroot builder.